main.js

```diff
@@ -0,0 +1,26 @@
+var $car = document.querySelector('.car>img');
+function turnCarUp(car) {
+  car.className = 'face-up';
+}
+function turnCarDown(car) {
+  car.className = 'face-down';
+}
+function turnCarRight(car) {
+  car.className = 'face-right';
+}
+function turnCarLeft(car) {
+  car.className = 'face-left';
+}
```

Comment:
Instead of having an individual function for each direction, you should have this accept two parameters: the car and the className for the direction